docs/en/intro-to-loom.md

```diff
@@ -12,10 +12,10 @@ In short, you can think of Loom Network as EOS on top of Ethereum.
 
 For advanced teams that need a custom blockchain, the Loom SDK will also let you [spin up](basic-install-all.html) your own DPoS chain in just a few minutes.
 
-## What is PlasmaChain?
+## What is Basechain?
 
-PlasmaChain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
+Basechain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
```

Comment:
[This content is getting a complete rewrite: https://github.com/loomnetwork/loom-sdk-documentation/pull/373]